src/os/os.c

```diff
@@ -135,7 +135,7 @@ typedef struct task_list
 } TaskList;
 
 /** List of all the tasks in the system */
-static TaskList taskList;
+static TaskList *taskList;
```

Comment:
= NULL;
(the previous decl was default constructed, this is not as declared here)